core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -389,15 +389,7 @@ public CacheManager.State getCacheManagerState() {
   public void moveBlock(long sessionId, long blockId, AllocateOptions moveOptions)
       throws IOException {
     // TODO(bowen): implement actual move and replace placeholder values
-    int dirIndex = getDirIndexOfBlock(blockId);
-    BlockStoreLocation srcLocation = new BlockStoreLocation(DEFAULT_TIER, dirIndex);
-    BlockStoreLocation destLocation = moveOptions.getLocation();
```

Comment:
the destination location is empty `alluxio.worker.block.BlockStoreLocation#ANY_MEDIUM`, and is used by the `moveBlock` operation as a special place holder. It's not supposed to be the actual block location.